### ISSUE TITLE
AUR: verify install

### ISSUE THREAD

**Issue by dsche-cyber** [bisq-network/bisq#4326]
### Description  
I see a tar.gz, but no tar.gz.asc.
So can't verify, when building myself.
AUR bisq seems also not checking the asc or sig. It just gives a sha256sum, which I don't find in the repository.

In the PKGBUILD I find:
source=("bisq.desktop")
sha256sums=('687d643fbe84660c3ebfe6270de98214f2e3ea791cb1d07d96d7ed889d61d406')

Can somebody help to verify bisq running on archlinux?

#### Version

1.3.5

### Expected behaviour
verifiable tar.gz or sha sum of aur package

### Actual behaviour
Not verifiable

#### Device or machine
5.6.16-1-Manjaro

**Comment by freimair** [comment]
I did the sha256sum-sums of the source tarball by myself after I verified the build manually.

And then, we switched to git lfs for the binary files and github does not provide the actual files in the source tarball, which is a bummer and was quite surprising. So we had to switch back to cloning the repo to make the AUR bisq package work again.

Not sure what to do at this point. Its on my list. Any suggestions?

EDIT: ah yes, the tar.gz on the release page is created by github - that is why there is no signature file and no binaries. Providing that manually is probably the correct way to address this issue.
